client/ladder/ladder.tsx

```diff
@@ -239,6 +241,13 @@ const PlayerCell = styled(BaseCell)`
   align-items: center;
 `
 
+const RaceCell = styled(BaseCell)`
+  width: 48px;
+  display: flex;
+  justify-content: flex-end;
```

Comment:
end alignment is reserved for numeric-ish data, I don't think races really qualify as that